packages/devtools_app/test/support/devtools_server_driver.dart

```diff
@@ -38,10 +38,20 @@ class DevToolsServerDriver {
   static Future<DevToolsServerDriver> create() async {
     // These tests assume that the devtools package is present in a sibling
     // directory of the devtools_app package.
-    final Process process = await Process.start(
-      Platform.resolvedExecutable,
-      <String>['../devtools/bin/devtools.dart', '--machine', '--port', '0'],
-    );
+    final args = <String>[
```

Comment:
You should be able to remove the `<String> `type here.

Reply:
Done!